.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description of change
+
+<!-- Briefly describe your changes here. If you are replacing an existing icon, provide a before and after screenshot -->
+
+### Link to demonstration
+
+<!-- Provide a link to a interactive map that uses the new icon with 11x11 and 15x15 variation. -->
```

Comment:
I know we already chatted about this in the meeting, but removing the 11x11 and 15x15 here, and then updating to what we ultimately decide on (I am not sure if we agreed on 18x18 or 24x24 on Friday)